nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -86,6 +86,8 @@
 
 - `services.outline.sequelizeArguments` has been removed, as `outline` no longer executes database migrations via the `sequelize` cli.
 
+- The binary of the package `cloud-sql-proxy` has changed from `cloud_sql_proxy` to `cloud-sql-proxy`.
```

Comment:
At first this felt a bit small for a rl entry but we already have a precedent  in rl-2111.section.md at line 418 so I guess it's fine.

Reply:
Yeah, it could after all be a breaking change for some and like you said, there already were examples in the release notes.

Reply:
yeah it was more meant as a note for other people looking at the pr ^^